health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/Attendance.js

```
@@ -0,0 +1,339 @@
+          paginationPerPage={rowsPerPage}
+          sortIcon={<CustomSVG.SortUp width="16px" height="16px" fill="#0b4b66" />}
+          paginationRowsPerPageOptions={[5, 10, 15, 20]}
+          noDataComponent={<div style={{padding: "24px"}}>{t("NO_RECORDS_FOUND")}</div>}
+        />
+
+        {assignUsersPopup && (
+          <PopUp
+            className="roleComposer"
+            type="default"
+            heading={t(`ASSIGN_USERS_TO_SUPERVISOR ${curSupervisor}`)}
+            children={[
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)